src/react-guides/Guides.tsx

```diff
@@ -1,10 +1,10 @@
-import Ruler, { PROPERTIES as RULER_PROPERTIES, RulerProps } from '@scena/react-ruler';
 import autobind from 'autobind-decorator';
 import { calculateMatrixDist, getDistElementMatrix } from 'css-to-mat';
 import Gesto, { OnDrag, OnDragEnd } from 'gesto';
 import * as React from 'react';
 import styled, { StyledElement } from 'react-css-styled';
 
+import Ruler, { PROPERTIES as RULER_PROPERTIES, RulerProps } from '../react-ruler';
```

Comment:
Why we do such weird thins ? 
`PROPERTIES as RULER_PROPERTIES`
can we import type as correct type immediately?

Reply:
Because we import Properties to the Guides class from the Ruler class.